src/SignupForm.js

```diff
@@ -4,6 +4,17 @@ import './SignupForm.css';
 
 const getAjaxUrl = url => url.replace('/post?', '/post-json?')
 
+function formatLinks(msg) {
+  let dom = document.createRange().createContextualFragment(msg);
+  var links = dom.querySelectorAll("a");
+  links.forEach(function(link) {
+    link.setAttribute('target', '_blank');
+  });
```

Comment:
so the problem is that in internet explorer, `forEach` is not available on the results of dom queries. `querySelectorAll` returns a `NodeList`, which isn't strictly an Array. so to make this safer, try this:
```js
for (let i = 0; i++; i < links.length) {
  links[i].setAttribute('target', '_blank');
}
```
babel should take care of that `let` statement, but I don't think it will polyfill `NodeList.forEach`.
More here: https://developer.mozilla.org/en-US/docs/Web/API/NodeList

Reply:
thanks, done